### DISCUSSION TITLE
Ultima template search bar partially hidden when menu is fixed in reveal style

### DISCUSSION THREAD

**Discussion by ahallez** [primefaces/community#2602] [PrimeVue Templates]
I found following issue in the Ultima template:
Select meny style 'reveal' and then fix the menu to remain visible.
The topbar and the content are nicely resized, but the search bar is not. So if you click the search icon, the search bar is partially hidden under the sidebar.

**Comment by tugcekucukoglu** [comment]
Thank you. I'll check.
